social/app/models/author.py

```diff
@@ -103,7 +101,15 @@ def create_profile(sender, **kwargs):
         user_profile.node = Node.objects.get(local=True)
         user_profile.save()
 
+def update_profile(sender, **kwargs):
+    from social.tasks import get_github_activity
+    user = kwargs["instance"]
+    if not user.is_staff:
+        user_profile = Author(user=user)
+        if user_profile.github != "":
+            get_github_activity(user_profile.id)
 
 post_save.connect(create_profile, sender=User)
+post_save.connect(update_profile, sender=User)
```

Comment:
The sender isn't User, it's Author, right? The hook you set up will run when a User gets updated, not an Author.